### DISCUSSION TITLE
"email notifications" translation

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11102] [Archive]
How to submit "`email notifications`" translation when there is no Crowdin such string?
It looks like that the translations are based on "`email`" and "`notifications`" separately.
But in some languages it should at least be written "`notifications email`" in the reverse order.

**Comment by rijkvanzanten** [comment]
Where is this string used?

**Reply by Pachat** [reply]
It is used in the Directus User's Detail where it cannot be parametrized.

**Reply by rijkvanzanten** [reply]
Ah gotcha. Pretty sure that's actually because it's just the title-formatted version of the column name `email_notifications` 🙂 

Lemme push up a tweak that makes it a properly translated string 👍🏻

**Reply by rijkvanzanten** [reply]
See https://github.com/directus/directus/pull/11103 👍🏻
